src/main/java/org/sciborgs1155/robot/Shooter/FlyWheel.java

```diff
@@ -0,0 +1,54 @@
+package org.sciborgs1155.robot.Shooter;
```

Comment:
Wow, you like wrote a whole robot! That's super impressive. Since we're merging everyone's code, get rid of all the shooter (and drive) stuff for now, but if you want to save it somewhere else that's fine, and I can do a review of it another time if you want.